From 7ca89c75b90f7d1972eed500168ae5c6ca064f36 Mon Sep 17 00:00:00 2001
From: Tung Vo <tung.vo.eb@renesas.com>
Date: Mon, 11 Mar 2019 19:50:09 +0700
Subject: [PATCH 140/175] arm64: dts: renesas: r8a774c0: Add USB3.0 function
 device nodes

Add usb3.0 function device nodes to the RZ/G2E SoC dtsi.

Signed-off-by: Tung Vo <tung.vo.eb@renesas.com>
---
 arch/arm64/boot/dts/renesas/r8a774c0.dtsi | 11 +++++++++++
 1 file changed, 11 insertions(+)

diff --git a/arch/arm64/boot/dts/renesas/r8a774c0.dtsi b/arch/arm64/boot/dts/renesas/r8a774c0.dtsi
index 3006087..9572146 100644
--- a/arch/arm64/boot/dts/renesas/r8a774c0.dtsi
+++ b/arch/arm64/boot/dts/renesas/r8a774c0.dtsi
@@ -1405,5 +1405,16 @@
 			#size-cells = <0>;
 			status = "disabled";
 		};
+
+		usb3_peri0: usb@ee020000 {
+			compatible = "renesas,r8a774c0-usb3-peri",
+				     "renesas,rcar-gen3-usb3-peri";
+			reg = <0 0xee020000 0 0x400>;
+			interrupts = <GIC_SPI 104 IRQ_TYPE_LEVEL_HIGH>;
+			clocks = <&cpg CPG_MOD 328>;
+			power-domains = <&sysc R8A774C0_PD_ALWAYS_ON>;
+			resets = <&cpg 328>;
+			status = "disabled";
+		};
 	};
 };
-- 
2.7.4

